src/sidebar/components/PendingUpdatesNotification.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button, RefreshIcon } from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+import { useCallback, useEffect, useRef, useState } from 'preact/hooks';
+
+import { useShortcut } from '../../shared/shortcut';
+import { withServices } from '../service-context';
+import type { StreamerService } from '../services/streamer';
+import { useSidebarStore } from '../store';
+
+export type PendingUpdatesNotificationProps = {
+  // Injected
+  streamer: StreamerService;
+
+  // Test seams
+  setTimeout_?: typeof setTimeout;
+  clearTimeout_?: typeof clearTimeout;
+};
+
+const collapseDelay = 5000;
+
+function PendingUpdatesNotification({
+  streamer,
+  /* istanbul ignore next - test seam */
+  setTimeout_ = setTimeout,
+  /* istanbul ignore next - test seam */
+  clearTimeout_ = clearTimeout,
+}: PendingUpdatesNotificationProps) {
+  const store = useSidebarStore();
+  const pendingUpdateCount = store.pendingUpdateCount();
+  const hasPendingUpdates = store.hasPendingUpdates();
+  const applyPendingUpdates = useCallback(
+    () => streamer.applyPendingUpdates(),
+    [streamer],
+  );
+  const [showCollapsed, setShowCollapsed] = useState(false);
```

Comment:
```suggestion
  const [collapsed, setCollapsed] = useState(false);
```

I think the addition of "show" in front makes the meaning less clear.